templates/README.md

```diff
@@ -0,0 +1,88 @@
+# Templates :pencil:
+
+We have prepared for you some templates. User templates allow you to apply predefined settings to start building your own applications. Using templates, you can easily set up multiple applications with similar settings.
+
+You can use it as a base for your own samples 👉 [sample template](https://github.com/scramjetorg/scramjet-cloud-docs/tree/main/templates). For this moment we support two variants of template in two programming languages:
+ - JavaScript (Node.js) 👉 [template](https://github.com/scramjetorg/scramjet-cloud-docs/tree/main/templates/template-js)
+ - TypeScript (ts-node) 👉 [template](https://github.com/scramjetorg/scramjet-cloud-docs/tree/main/templates/template-ts)
+
+In the following sections we will describe both templates' contents and show you how they can be used in your own project.
+
+## Work with JavaScript (Node.js)
+
+This application package template contains two files:
+
+- package.json - JSON manifest file describing the app and its configuration such as main file to run. This is one of the Node.js project's basic requirement to have the `package.json` file in the root of every project. It contains metadata information in plain JSON Object format. This is the heart of any Node.js project It is used by the application to find its dependencies to install, scripts to run, etc. If you would like to start your project from scratch, without using our js-template, you would need to create a directory and run `npm init` inside. This command would create a `package.json` file for you with all the obligatory properties, for example:
```

Comment:
I don't know why, but `JSON manifest file` reminds me of Progressive Web App where you can find manifest.json: https://web.dev/add-manifest/.

"The package.json file is kind of a manifest for your project." => JSON is like a ...?

Reply:
An interesting observation. Now that I think about it, the manifest reminds me of the famous https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar :)

Reply:
fixed